gen/tests/test_adminrouter_tls_conf.py

```diff
@@ -12,15 +12,18 @@ class TestAdminRouterTLSConfig:
     Tests for the Admin Router TLS Config creation.
     """
 
```

Comment:
This is missing test cases for the defaults. As Martin is away, I will create a JIRA issue.